pkg/resourcecreator/ingress/ingress.go

```diff
@@ -175,10 +177,16 @@ func nginxIngresses(source Source, cfg Config) ([]*networkingv1.Ingress, error)
 	ingresses := make(map[string]*networkingv1.Ingress)
 
 	for _, rule := range rules {
-		ingressClass := util.ResolveIngressClass(rule.Host, cfg.GetGatewayMappings())
+		var ingressClass *string
+
+		if len(cfg.GetGatewayMappings()) > 0 {
```

Comment:
this will never happen because of
https://github.com/nais/naiserator/pull/343/files#diff-4e0974ccfedc134af693fcb54c0a37cd5e3e6d0945412f04e8dac7b80e9beaf8R129

Reply:
Good point! :) Removing check...